### PR TITLE
Adapt iframe and image width when in right panel

### LINKED ISSUE #184: iframes in the sidebar are too small
Originally reported by: **Anonymous**

---

If you open an objekt on my map:
http://umap.openstreetmap.fr/de/map/wikipedia-clustermap_36725#14/51.1440/11.9136
you will see that I load the mobile Wikipedia page into an iframe. The Iframe is too small an right of it is a lot of wasted space. 

Greetings Kolossos 

---
- Bitbucket: https://bitbucket.org/yohanboniface/umap/issue/184

### PR DESCRIPTION
**Pull request by yohanboniface**
fix #184

Before:
Screenshot-20200317194354-1656x1882

After:
Screenshot-20200317194403-1846x1886

### CODE REVIEW

File: umap/static/umap/map.css [hunk 1]
```diff
@@ -1304,6 +1304,10 @@ a.add-datalayer:hover,
     /* See https://github.com/Leaflet/Leaflet/commit/61d746818b99d362108545c151a27f09d60960ee#commitcomment-6061847 */
     max-width: 500px !important;
 }
+#umap-ui-container .umap-popup-content img,
+#umap-ui-container .umap-popup-content iframe {
+    max-width: 400px !important;
```
Comment (yohanboniface) [inline umap/static/umap/map.css]: `400px` is the `#umap-ui-container` width (defined in `base.css`).

**Comment by Binnette** [comment]
Hi, Is your "test" map public ? I want to reproduce the "bug" 😉

**Comment by yohanboniface** [comment]
Ah, may interfere with https://github.com/umap-project/umap/issues/188 :/

**Comment by Binnette** [comment]
\o/ I finally made it !!!

image

And it is fully responsive 👍 👍 👍 
image

**Comment by yohanboniface** [comment]
:+1: 

I'll let you push on that PR ?

**Comment by Binnette** [comment]
Here is the way to do so :
1. [optional but must have!] set the bottom padding of #umap-ui-container to 0px
image
2. Delete those annoying min-with/max-width from ".......... iframe" classes
image
3. I rewrite part of the div umap-popup
- I deleted the useless div with class=""
- I set umap-content-popup to display:flex with flex-direction: column
- I put h3, br and a in a new div
- I removed almost all attributes on the iframe and set style instead. 
image

TADA !!!

**Comment by Binnette** [comment]
Do you want to do it in order to understand my "hack"? Otherwise, i can push on this PR 👍

**Comment by yohanboniface** [comment]
You can push on this branch directly :)

**Comment by Binnette** [comment]
Ok, i tried to fix code that is not in umap :stuck_out_tongue_winking_eye:

https://tools.wmflabs.org/wp-world/marks-umap.php?LANG=en&coats=0&thumbs=0&bbox=7.750028371810914,48.57887501718689,7.7603280544281015,48.58108601867744&maxRows=80&ttl=
image

**Comment by yohanboniface** [comment]
This is how to set the height of the iframe.

**Comment by Binnette** [comment]
Ok this time everything is fine :smile: Can you please test it?

Panel mode:
image

Popup mode (normal and large):
image

**Pull request merged**